src/commands/Shop.js

```diff
@@ -1,13 +1,22 @@
+/** @typedef {import('discord.js/src/structures/Message')} Message */
+
 import BaseCommand from './BaseCommand.js';
 
 export default class Shop extends BaseCommand {
+    /**
+     *
+     */
     constructor() {
         super({
-            aliases: ['shop', 'sh']
-        })
+            aliases: ['shop', 'sh'],
```

Comment:
Pass `aliases` as separate param. See `BaseCommand` constructor params in `BaseCommand.js` line 8 .